bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WWWJets_4f/WWWJets_4f_NLO_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model loop_sm-no_b_mass
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define w = w+ w-
+generate p p > w w w $$ t t~ [QCD] @0
```

Comment:
this tri-w sample is a fully inclusive one? again i think you can use madspin

Reply:
set ms_dir ./madspingrid
set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
set max_running_process 1
decay t > w+ b, w+ > all all
decay t~ > w- b~, w- > all all
decay w+ > all all
decay w- > all all
launch

this madspin card will work?

Reply:
same here